setupContents.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+func externalContents() []byte {
```

Comment:
Why use a function here? These could be constants easily enough. Also, `string` is usually easier to work with than `[]byte`. I typically only convert my `string`s to `[]byte`s when writing them to files.

Reply:
Done.